packages/hardhat-core/src/internal/sentry/anonymizer.ts

```diff
@@ -226,4 +234,80 @@ export class Anonymizer {
 
     return result;
   }
+
+  private _anonymizeMnemonic(
+    errorMessage: string,
+    mnemonicPhraseLengthThreshold: number,
+    minimumAmountOfWordsToAnonymize: number
+  ): string {
+    const fragments = errorMessage.split(/[!$%^&*()_+|~\-=`{}[\]:;<>?"',.]/gu);
+    const fragmentsToBeAnonymized: string[] = [];
+    // If there are 10 consecutive words, there's a good chance of there being a mnemonic phrase
```

Comment:
what's the 10 here?

Reply:
Hm, that's old. It's a reference to the `mnemonicPhraseLengthThreshold` constant.